concepts/60 Themes and Styles/05 Predefined Themes/50 Apply a Theme.md

```diff
@@ -1,9 +1,9 @@
 Do the following to apply a theme:
 
-- **jQuery, Knockout, AngularJS**   
-Include `dx.common.css` and a theme stylesheet in the `<head>` tag on your index page. You can use [local files](/concepts/Common/Distribution%20Channels/15%20ZIP%20Archive.md '/Documentation/Guide/Common/Distribution_Channels/ZIP_Archive/') or get the stylesheets [from CDN](/concepts/Common/Distribution%20Channels/05%20CDN '/Documentation/Guide/Common/Distribution_Channels/CDN/').
+- **jQuery**   
+Include `dx.common.css` and a theme stylesheet in the `<head>` tag on your index page. You can use [local files](/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/#Local_Files) or get the stylesheets [from CDN](/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/#CDN_Files).
```

Comment:
Can we say:
Include `dx.common.css` and a theme stylesheet in the index page's `<head>` tag. You can use [local files](/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/#Local_Files) or get stylesheets [from CDN](/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/#CDN_Files).